testing/smoke-test/build-pack.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Smoke test environment setup script
+
+cu_p=packages/core-util
+d_p=packages/dashboard
+rc_p=packages/react-components
+sw_p=packages/source-iotsitewise
```

Comment:
dont we need source_twinmaker and scene composer as well, even not used in dashboards they are in react-components.